packages/button/README.md

```diff
@@ -1,27 +1,20 @@
 #  [`@fremtind/jkl-button`](https://fremtind.github.io/jokul/components/button/)
 
-Knapper starter en handling. De oppfordrer brukeren til å gjøre noe. Teksten på knappen forteller hva som vil skje når brukeren velger den.
+Knapper starter en handling. De oppfordrer brukeren til å gjøre noe. Teksten på knappen forteller hva som vil skje når brukeren velger den. Våre knapper bruker elevasjon for å trekke fokus, når brukeren holder pekeren over eller velger en knapp blir den løftet fra siden (forstørret).
```

Comment:
Kan vi skrive på en enklere måte? F. eks: "For å gi fokus forstørres knappen når brukeren holder pekeren over eller velger"

Reply:
Oppdatert i nyeste